tests/compiler.js

```diff
@@ -1125,5 +1125,35 @@
                   'may the forth be with you');
             finish(done);
         });
+        
+
+        it('should thrown an error', function(done) {
+            render(
+                '{% include "undefined-macro.html" %}',
+                {},
+                { noThrow: true },
+                function(err, res) {
+                    expect(res).to.be(undefined);
+                    expect(err).to.match(/Unable to call `\w+`, which is undefined or falsey/);
+                }
+            );
+
+            finish(done);
+        });
+
+        it('should thrown an error', function(done) {
```

Comment:
And again here. Definitely don't want two cases with exactly the same description; makes the test-runner output much less useful if one of them fails.